src/components/Missions/Missions.jsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import Mission from './Mission';
+import { fetchMissions } from '../../redux/Missions/Missions';
```

Comment:
- [ ] Nice work done on your Mission component, kindly check this import `../../redux/Missions/Missions`, I had issues starting your project because the name of the folder starts with a capital `R`edux and you imported it using a Lowercase `r`edux